AuxTel/Non-Standard-Operations/Recovery-after-Shutdown/Recovery-after-Shutdown.rst

```diff
@@ -0,0 +1,196 @@
+2. If the WREB board has not been powered up, then ats-fp will not be reporting.  This requires starting up the WREB board with the ats-init.py script, followed by turning on the HV bias.   Detailed instructions for doing this are in   https://sitcomtn-026.lsst.io/
+3. Sometimes, bonn-shutter has a fault which can not be cleared in this way.  When this happens, the only way that has been found to clear this is to physically power cycle the shutter controller.  Figure 11 shows the location of the bonn shutter controller Power cycle it by unplugging the power cable, waiting a few seconds, and plugging it back in.  This usually clears the fault.  Again, I think it is possible to do this remotely by logging into the PDU, but I have never done this.
+
+   .. figure:: /AuxTel/Non-Standard-Operations/_static/Shutter_reboot.jpg
+Figure 11: Power cycling the bonn-shutter controller.
+
+Step 3 - Bringing ats-ocs-bridge to the proper state.
+++++++++++++++++++++++++++++++++++++++++++++++++++
+
+One of the CCS modules is ats-ocs-bridge.  This is the subsystem that interfaces between CCS and the Observatory Control System (i.e. the CSCs).  In this case ats-ocs-bridge is interfacing with the ATCamera CSC.  It is necessary to get ats-ocs-bridge into the proper state in order to be able to control ATCamera with LOVE and the script queue.  Here are the necessary steps:
+
+1. Run the following ccs-shell command: >ats-ocs-bridge getState
+ This will return something like::
+   
+   AlertState:NOMINAL CCSCommandState:IDLE CommandState:READY ConfigurationState:CONFIGURED OfflineState:OFFLINE_PUBLISH_ONLY OperationalState:ENGINEERING_OK PhaseState:OPERATIONAL SummaryState:OFFLINE           
+
+2. The SummaryState is the same state of ATCamera you see with LOVE.  If the SummaryState is FAULT, it cannot be brought out of fault with the normal LOVE commands.  It needs to be brought out of fault with the ccs-shell command >ats-ocs-bridge clearFault command.
+
+3. Assuming the SummaryState is OFFLINE, then we look at the OfflineState.  If the OfflineState is:OFFLINE_PUBLISH_ONLY, we need to transition it to OFFLINE_AVAILABLE before we can use the usual state transition commands in LOVE and the script queue to bring it online.  This is done with the ccs-shell command >ats-ocs-bridge setAvailable.
+
+4. Once we have it in SummaryState OFFLINE and OfflineState OFFLINE_AVAILABLE, we can then transition it using the script queue and the set_summary_state.py to bring the SummaryState to STANDBY.  I am assuming you know how to do this.
+
+5. Once the SummaryState is STANDBY, you can run the 'start' and 'enable' commands to bring up ATCamera.  Or you can run enable_latiss.py in the script queue to bring up all of LATISS.  If this is successful, things should now be operating normally.
```

Comment:
I suggest to add here to make a clean standby-enable cycle on ATCS and LATISS  (auxtel/standby_atcs   ---> auxtel/enable_atcs, auxtel/standby_latiss  ---> auxtel/enable_latiss).